src/XenobladeChroniclesX/Mods/BattleDamageModGround2/rules.txt

```diff
@@ -0,0 +1,28 @@
+[Definition]
+titleIds = 0005000010116100,00050000101C4C00,00050000101C4D00
+name = "Ground damage divisor
+path = "Xenoblade Chronicles X/Mods/Battle/Ground damage divisor
+description = Decrease the ground damage done by your team. I wanted to increase the challenge of the main story but the Enemy Stat Multiplier Mod made them hit way too hard. Note: displayed damage isn't affected so in practice it feels more like multiplying the enemy's health, but internally it's really the damage which is divided. Note 2: Haven't tested it for skells, but it probably won't work since the Damage Mod Multiplier doesn't. Note 3: I'm no /u/lasyan3 and frankly I had no idea what I was doing lol. Unexpected behaviour might show up. Nothing happened to me so far though.
```

Comment:
This description is pretty long. Just saying that it's experimental would probably be enough. Preferably it shouldn't use "I" since it's not really clear for normal users as to who that is referring to. I'd personally change it to
```suggestion
description = Make the game more challenging by dividing the given damage you and your team do to enemies. Doesn't affect the damage numbers that are shown.|Might be experimental since it's not fully been tested yet.||Made by Lashoun.
```

Reply:
Fine by me!